app/src/main/java/com/android/sample/model/userAccount/UserAccountViewModel.kt

```diff
@@ -163,6 +165,7 @@ open class UserAccountViewModel(
   }
 
   fun rejectFriendRequest(friendId: String) {
+    updateReceivedListAfterRemoval(friendId)
```

Comment:
Same here I think it might be better to place it in the onSuccess part

Reply:
Same answer.